.github/workflows/docker.yml

```diff
@@ -4,8 +4,16 @@ name: Master docker CI
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/pull_request_template.md'
+      - 'CONDE_OF_CUNDUCT.md'
```

Comment:
CONDE OF CUNDUCT?!?!?!?

Reply:
`CODE_OF_CONDUCT.md`

Reply:
Shh.
Forgot to fix that file :/
type fast bad